src/pages/home/report/ReportActionsList.js

```diff
@@ -222,20 +222,28 @@ function ReportActionsList({
     }, [report.lastReadTime, report.reportID]);
 
     useEffect(() => {
-        // If the reportID changes, we reset the userActiveSince to null, we need to do it because
-        // this component doesn't unmount when the reportID changes
```

Comment:
Have you tested this case the comment mentions, i.e. when the "reportID changes"?

I believe the changes you suggested are clean and improve the code, but I just want to be sure we don't introduce a regression, even when the intentions are pure